### DISCUSSION TITLE
With Mac no basemap is displayed in QField

### DISCUSSION THREAD

**Discussion by kerstin8** [opengisch/QField#3883] [Q&A]
Hi Everyone, I am working with a Macbook and an iPad 12.9. When I open a project in QField, I am not shown the basemap. A punklayer works. In QField I have a project folder Test_qfield. And the following datasets: basemap.gpkg, Test.zip, and Data.gpkg. After opening the project I have a white page in front of me. When the project is loaded on an Android device it works. Does anyone have an idea if it is due to a setting?
I have already made a phone call to Apple. They say that it must be related to QField.

translated with deepl free version

**Comment by nirvn** [comment]
@kerstin8 , if you basemap relies on JPG (for e.g. a JPG tiled basemap), it will not work on iOS. This is a known issue, you can follow updates on this here: https://github.com/opengisch/QField/issues/3611

**Comment by kerstin8** [comment]
Dear Mathieu,

thank you very much for your quick reply.
I am glad that the problem is known. I am a QGis beginner and I have to search how to create a PNG grid with a PNW world file. I assume that it worked for you. :-)

First of all thanks for your tip!
Have a nice day!
Kerstin
